src/commands/delete-organization-member.ts

```diff
@@ -0,0 +1,54 @@
+import {BaseCommand} from '../lib'
+import {getEnvironment} from '../lib/environments'
+import {flags} from '@oclif/command'
+
+const DELETE_ORGANIZATION_MEMBER = `
+mutation DeleteOrganizationMember($member: DeleteOrgMember!) {
+  deleteOrganizationMember(input: $member) {
+      uid
+      name
+  }
+}
+
+`
+
+export default class DeleteOrganizationMember extends BaseCommand {
+  static description = 'Delete a Member from Organization'
+
+  static examples = [
+    '$ slash-graphql delete-organization-member 0x123 -m 0x1234',
```

Comment:
don't we have an API that removes the member by email? This is weird that you add by email, but remove by uid. Can you make this email too?

Reply:
We don't have an API that removes the member by email.